docs/src/diagrammer.js

```diff
@@ -0,0 +1,11 @@
+const parserFactory = require('grammar');
+const diagram = require('chevrotain/diagrams/src/main.js');
+
+
+const Parser = parserFactory.default('[Comment]', '[EndComment]').Parser;
+const parserInstanceToDraw = new Parser([]);
+
+
+module.exports = function(targetDiv) {
```

Comment:
??

All the interactivity seems to work:
http://htmlpreview.github.io/?https://raw.githubusercontent.com/kevinastone/atom-grammar-test/pack-diagram/docs/grammar.html